commit 64e9452e2184ed6fc05de0025ff91133f0874196 (HEAD)
Author: Adam Kallai <kadam@inf.u-szeged.hu>
Date:   Fri Jul 30 18:15:42 2021 +0000

    Fix the build on Windows on ARM
    
    Extend the installer logic with ARM processor architecture,
    in order to make the win-arm64 bot happier.
    
    This CL doesn't cover its tests.
    
    Bug: chromium:1233065
    Change-Id: I2d50a801e0d020eb7d66be85e17df04b417e11e5
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3056003
    Reviewed-by: Yann Dago <ydago@chromium.org>
    Reviewed-by: Gabriel Charette <gab@chromium.org>
    Commit-Queue: Richard Townsend <richard.townsend@arm.com>
    Cr-Commit-Position: refs/heads/master@{#907186}

    ckerr comment for Electron: Cherry-picked as a short-term patch because this
    seems to be the last blocker before the main roll is green. The roll is so far
    behind that  I'd rather bank this savepoint than gamble on other issues arising
    from a deps bump. Can be removed as soon as the roll catches up.

diff --git a/chrome/installer/util/additional_parameters.cc b/chrome/installer/util/additional_parameters.cc
index 275c635154111..ddddd5bbfb405 100644
--- a/chrome/installer/util/additional_parameters.cc
+++ b/chrome/installer/util/additional_parameters.cc
@@ -201,6 +201,8 @@ std::wstring GetChannelIdentifier(version_info::Channel channel,
   static constexpr base::WStringPiece kArchSuffix = L"-arch_x64";
 #elif defined(ARCH_CPU_X86)
   static constexpr base::WStringPiece kArchSuffix = L"-arch_x86";
+#elif defined(ARCH_CPU_ARM64)
+  static constexpr base::WStringPiece kArchSuffix = L"-arch_arm64";
 #else
 #error unsupported processor architecture.
 #endif
@@ -236,6 +238,8 @@ std::wstring GetChannelIdentifier(version_info::Channel channel,
       return L"x64-stable";
 #elif defined(ARCH_CPU_X86)
       return L"stable-arch_x86";
+#elif defined(ARCH_CPU_ARM64)
+      return L"stable-arch_arm64";
 #else
 #error unsupported processor architecture.
 #endif
